README.md

```diff
@@ -44,49 +50,6 @@ For example if you want to work with json/csv files here are dependencies you wi
 composer require flow-php/etl:^0.1 flow-php/etl-adapter-csv:^0.1 flow-php/etl-adapter-json:^0.1
 ```
 
-## Contributing 
-
-In order to run tests locally please make sure you have [docker](https://www.docker.com/) up and running.
-You also need [PHP 8.1](https://www.php.net/) and [composer](https://getcomposer.org/) to be available from your CLI.
-
-For the code coverage, please install [pcov](https://pecl.php.net/package/pcov).
-
-### Prepare Project:
-
-```shell
-cp docker-compose.yml.dist docker-compose.yml
-composer install 
-docker compose up -d
-```
-
-### Run Test Suite 
-
-```shell
-composer test
-```
-
-### Run Static Analyze 
-
-```shell
-composer static:analyze
-```
-
-### Test everything
-
-This command will execute exactly the same tests as we run at Github Actions before PR can get merged.
-If it passes locally, you are good to open pull request. 
-
-```shell
-composer build 
-```
-
-## Building PHAR
-
-```shell
-composer build:phar
-./flow-php.phar --version
-```
-
```

Comment:
I would move `Features` at the top, before going into details about how to install etc.